### PR TITLE
Adding unit test for coverage

### PR DESCRIPTION
**Pull request by kuldeepk3**
Change log description
Adding unit test for coverage.

Purpose of the change
Improvement of code coverage.

What the code does?
Adds unit tests to improve code coverage

How to verify it
Building the project will gives improved coverage

### CODE REVIEW

File: pravega-sensor-collector/src/test/java/io/pravega/sensor/collector/file/parquet/ParquetFileIngestServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+
+    class TestParquetFileIngestService extends ParquetFileIngestService {
+
+        public TestParquetFileIngestService(DeviceDriverConfig config) {
+            super(config);
+        }
+
+        @Override
+        protected void createStream(String scopeName, String streamName) {
+            System.out.println("Create Stream do nothing");
```
Comment (sachin-j-joshi) [inline pravega-sensor-collector/src/test/java/io/pravega/sensor/collector/file/parquet/ParquetFileIngestServiceTest.java:100]: Please remove System.out.println. 
Also looks like the code is still under construction here.

File: pravega-sensor-collector/src/test/resources/RawFileIngestService.properties [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#PRAVEGA_SENSOR_COLLECTOR_RAW1_PRAVEGA_CONTROLLER_URI=tls://pravega-controller.texas-twister.ns.sdp.hop.lab.emc.com:443
+PRAVEGA_SENSOR_COLLECTOR_RAW1_PRAVEGA_CONTROLLER_URI=tcp://127.0.0.1:9090
+PRAVEGA_SENSOR_COLLECTOR_RAW1_SCOPE=test-psc
+PRAVEGA_SENSOR_COLLECTOR_RAW1_STREAM=test-psc-stream
+PRAVEGA_SENSOR_COLLECTOR_RAW1_ROUTING_KEY=$(hostname)
+PRAVEGA_SENSOR_COLLECTOR_RAW1_DELETE_COMPLETED_FILES=true
+PRAVEGA_SENSOR_COLLECTOR_RAW1_TRANSACTION_TIMEOUT_MINUTES=1.0
+PRAVEGA_SENSOR_COLLECTOR_RAW1_CREATE_SCOPE=true
+PRAVEGA_SENSOR_COLLECTOR_RAW1_MIN_TIME_IN_MILLIS_TO_UPDATE_FILE=5000
+PRAVEGA_SENSOR_COLLECTOR_RAW1_DELETE_COMPLETED_FILES_INTERVAL_IN_SECONDS=15
```
Comment (sachin-j-joshi) [inline pravega-sensor-collector/src/test/resources/RawFileIngestService.properties:26]: Please add newline.

File: pravega-sensor-collector/src/test/java/io/pravega/sensor/collector/file/rawfile/RawFileIngestServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+public class RawFileIngestServiceTest {
```
Comment (sachin-j-joshi) [inline pravega-sensor-collector/src/test/java/io/pravega/sensor/collector/file/rawfile/RawFileIngestServiceTest.java:23]: Looks like we can pull lots of common functionality into an abstract base class.
Both RawFileIngestServiceTest and ParquetFileIngestServiceTest can derive from it.

File: pravega-sensor-collector/src/test/java/io/pravega/sensor/collector/file/parquet/ParquetFileIngestServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,103 @@
...
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.nio.file.StandardCopyOption;
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+public class ParquetFileIngestServiceTest {
```
Comment (sachin-j-joshi) [inline pravega-sensor-collector/src/test/java/io/pravega/sensor/collector/file/parquet/ParquetFileIngestServiceTest.java:23]: Please add some basic comment on what this class tests.

**Review by sachin-j-joshi — CHANGES_REQUESTED** [review]
Please introduce a base class to avoid duplication of code.

File: pravega-sensor-collector/src/test/resources/RawFileIngestService.properties (hunk 2 above)
Comment (kuldeepk3) [reply pravega-sensor-collector/src/test/resources/RawFileIngestService.properties]: done

**Comment by kuldeepk3** [comment]
updated pr and removed unwanted code

File: pravega-sensor-collector/src/test/java/io/pravega/sensor/collector/file/MockFileIngestService.java [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(MockFileIngestService.class);
+    public MockFileIngestService(DeviceDriverConfig config) {
+        super(config);
+    }
+
+    /*
+     * Mocking the behaviour of create stream
+     */
+    @Override
```
Comment (sachin-j-joshi) [inline pravega-sensor-collector/src/test/java/io/pravega/sensor/collector/file/MockFileIngestService.java:17]: where is this defined in base class?

File: pravega-sensor-collector/src/test/java/io/pravega/sensor/collector/file/FileIngestServiceTest.java [hunk 6]
```diff
@@ -9,21 +9,56 @@
...
+    void setUp(){
+        properties = Parameters.getProperties(filename);
+        driverManager = new DeviceDriverManager(properties);
+        deviceDriverConfig = new DeviceDriverConfig("RAW1", "RawFileIngestService",
+                TestUtils.configFromProperties(PREFIX, SEPARATOR, properties), driverManager);
+    }
+
+    @Test
+    public void testFileIngestService(){
+        FileIngestService fileIngestService = new MockFileIngestService(deviceDriverConfig);
```
Comment (sachin-j-joshi) [inline pravega-sensor-collector/src/test/java/io/pravega/sensor/collector/file/FileIngestServiceTest.java:47]: Does it make sense to move this to `@Before` ?

**Review by sachin-j-joshi — APPROVED** [review]
LGTM

**Review by abhinb — APPROVED** [review]

**Pull request merged**